tests/profiling/collector/test_threading.py

```diff
@@ -537,6 +537,45 @@ def test_anonymous_lock():
     assert release_event.lock_name == "test_threading.py:%d" % release_lineno
 
 
+@pytest.mark.skipif(not os.getenv("WRAPT_DISABLE_EXTENSIONS"), reason="wrapt C extension is disabled")
```

Comment:
nit: Maybe these tests can be parameterized? Something like:
```python
@pytest.mark.parametrize(
   "wrapt_disable, init_lock",
   [
      (False, "545"),
      (True, "565")
)
def test_wrapt_c_ext_config(wrapt_disable, init_lock):
...
```

Reply:
Thanks for pointing that out. I just wanted to test that the results are the same regardless of the WRAPT_DISABLE_EXTENSIONS ENV var, and to be more precise regardless of `_lock.WRAPT_C_EXT` variable. So combined these two tests.